tests/test_units.py

```diff
@@ -20,203 +20,221 @@
-        'uV': [{'units': 'volt', 'prefix': 'micro'}],
-        'mM': [{'prefix': 'milli', 'units': 'mole'}, {'units': 'litre', 'exponent': '-1'}],
-        'milli_mole': [{'prefix': 'milli', 'units': 'mole'}],
-        'millisecond': [{'prefix': 'milli', 'units': 'second'}],
+        'ms': 'second / 1000',
+        'usec': 'second / 1000000',
+        'mV': 'volt / 1000',
+        'uV': 'volt / 1000000',
+        'mM': '(mole / 1000) / litre',
+        'milli_mole': 'mole / 1000',
+        'millisecond': 'second / 1000',
     })
     test_definitions.update({
-        'per_ms': [{'units': 'ms', 'exponent': '-1'}],
-        'per_mV': [{'units': 'volt', 'prefix': 'milli', 'exponent': '-1'}],
-        'mV_per_ms': [{'units': 'mV', 'exponent': '1'}, {'units': 'ms', 'exponent': '-1'}],
-        'mV_per_s': [{'units': 'mV', 'exponent': '1'}, {'units': 'second', 'exponent': '-1'}],
-        'mV_per_usec': [
-            {'units': 'mV', 'exponent': '1'}, {'prefix': 'micro', 'units': 'second', 'exponent': '-1'}],
-        'mM_per_ms': [{'units': 'mM'}, {'units': 'ms', 'exponent': '-1'}],
-        'ms_power_prefix': [{'prefix': '-3', 'units': 'second'}],
-        'ms_with_multiplier': [{'multiplier': 0.001, 'units': 'second'}],
+        'per_ms': '1 / ms',
+        'per_mV': '1 / mV',
+        'mV_per_ms': 'mV / ms',
+        'mV_per_s': 'mV / second',
+        'mV_per_usec': 'mV / usec',
+        'mM_per_ms': 'mM / ms',
     })
 
     @pytest.fixture(scope="class")
-    def quantity_store(self):
-        """ QuantityStore to capture units used in tests. """
-        qs = UnitStore(model=None)
-        for unit_name, unit_attributes in self.test_definitions.items():
-            qs.add_custom_unit(unit_name, unit_attributes)
-        return qs
+    def unit_store(self):
+        """A UnitStore for testing."""
+        s = UnitStore()
+        for name, definition in self.test_definitions.items():
+            s.add_unit(name, definition)
+        return s
 
-    def test_quantity_translation(self, quantity_store):
-        """ Tests that unit equality is correctly determined. """
-        unit_registry = quantity_store.ureg
-
-        assert quantity_store.get_quantity('dimensionless') == unit_registry.dimensionless
+    def test_quantity_translation(self, unit_store):
+        """Tests that unit equality is correctly determined."""
 
         # Units defined in the test CellML <model>:
         for name in TestUnits.test_definitions.keys():
-            assert quantity_store.get_quantity(name) != unit_registry.Quantity(1)
-
-        # Pint built-in units
-        assert quantity_store.is_unit_equal(
-            quantity_store.get_quantity('kilogram'),
-            unit_registry.kilogram
-        )
+            assert unit_store.is_unit_defined(name)
 
         # Custom units defined in CellML example
-        assert quantity_store.is_unit_equal(
-            quantity_store.get_quantity('per_ms'),
-            unit_registry.millisecond**(-1)
+        '''
```

Comment:
Are you meaning to skip these?

Reply:
If I can. They seem to be testing that `pint` works as intended. Happy to replace them if it turns out they're needed